flask_backend_design.py

```diff
@@ -0,0 +1,272 @@
+""" importing required libraries for this package """
+from flask import Flask, request, jsonify
+from flask import redirect, url_for
+import pyodbc
+import datetime
+import os
+date = datetime.datetime.now()
+CurrentUser = os.getlogin()
+
+app = Flask(__name__)
+
+""" Establishing connection to database"""
+
+
+def init_db():
+    con = pyodbc.connect("Driver={ODBC Driver 13 for SQL Server}; \
+                          SERVER=192.168.18.36;DATABASE=HBK_Test;\
+                          UID=maheshp;PWD=Welcome123")
+    cursor = con.cursor()
+    return cursor
+
+
+cursor = init_db()
+
+""" Validating the user details """
+
+
+@app.route('/user/<username>')
+def user(username):
+    user_object = get_user(username)
+    if type(user_object) is pyodbc.Row:
+        return redirect(url_for("get_customer_details", name=username))
+    else:
+        return "UserName is Invalid.Please open your account in CITI bank"
+
+
+""" Fetches user details from db"""
+
+
+def get_user(name):
+    cursor.execute("SELECT UserName,Password from USER_bankdemo with(Nolock) \
```

Comment:
All queries can be abstracted and moved to another module, say queries.py.
It would make code cleaner and readable. 
Trailing backslash.

Reply:
Separate out the API views from helper or utilities methods.